pyproject.toml

```diff
@@ -90,13 +85,32 @@ all = [
     "mpmath",
     "asdf-astropy>=0.3",
     "bottleneck",
-    "ipykernel",
-    "ipython>=7.32",
-    "ipywidgets",
-    "pytest>=7.0",
     "fsspec[http]>=2023.4.0",
     "s3fs>=2023.4.0",
+]
+# The base set of test-time dependencies.
+test = [
     "pre-commit",
+    "pytest>=7.0",
+    "pytest-doctestplus>=0.12",
+    "pytest-astropy-header>=0.2.1",
+    "pytest-astropy>=0.10",
+    "pytest-xdist",
+    "threadpoolctl",
+]
+test_all = [
+    # Install grouped optional dependencies
+    "astropy[all]",  # installs all optional run-time dependencies
+    "astropy[test]",
+    # Install all remaining dependencies
+    "objgraph",
+    "coverage[toml]",
```

Comment:
shouldn't this line be in `[test]` ? I also note that it's actually already there in a way (`pytest-astropy` requires `pytest-cov` which itself requires `coverage[toml]`)

Reply:
Moved!